cf-templates/xrd-eks-ami-cf.yaml

```diff
@@ -35,8 +35,13 @@ Parameters:
     Type: String
     Default: m5.2xlarge
 
+  SubnetId:
+    Description: Subnet where the instance building the AMI is launched. If left empty this uses the default subnet.
+    Type: String
```

Comment:
This can be `AWS::EC2::Subnet::Id` (which should result in a drop-down selection if launching via the console)